src/datajudge/requirements.py

```diff
@@ -166,8 +176,30 @@ def add_uniqueness_constraint(
         )
 
     def add_column_type_constraint(
-        self, column: str, column_type: str, name: str = None
+        self,
+        column: str,
+        column_type: Union[str, sa.types.TypeEngine],
```

Comment:
Why don't you make a type alias to this?

Reply:
I'm not sure if it would help with readability and understandability, there are only a handful of usages and it adds a layer of indirection needed when reading the documentation.